package.json

```diff
@@ -62,6 +62,8 @@
     "onlyPublishWithReleaseLabel": true
   },
   "dependencies": {
+    "@cliqz/adblocker": "https://gitpkg.vercel.app/muodov/adblocker/packages/adblocker?hiding-styles-hack",
```

Comment:
What's this hack? Is it something we should eventually upstream?

Reply:
It's a hack to change the default hiding snippet, see https://github.com/ghostery/adblocker/issues/4178
I have a couple TODO comments in web.ts referencing it. Generally yes, I hope to push it to upstream